packit/cli/update.py

```diff
@@ -59,6 +59,11 @@
     default=False,
     help="Upload the new sources also when the archive is already in the lookaside cache.",
 )
+@click.option(
+    "--no-pr",
+    is_flag=True,
+    help="Do not create a pull request to downstream repository.",
+)
```

Comment:
any default value ?

Reply:
I think it is the default (and not needed), but I made it explicit. I've add that also to the `downstream-to-upstream`...